frontend_tests/node_tests/composebox_typeahead.js

```diff
@@ -747,10 +763,14 @@ global.people.add(deactivated_user);
     assert_typeahead_equals("hi emoj:i", false);
     assert_typeahead_equals("hi emoji :ta", emoji_list);
     assert_typeahead_equals("hi emoji :da", emoji_list);
+    assert_typeahead_equals("hi emoji :da_", emoji_list);
+    assert_typeahead_equals("hi emoji :da ", emoji_list);
     assert_typeahead_equals("hi emoji\n:da", emoji_list);
     assert_typeahead_equals("hi emoji\n :ra", emoji_list);
     assert_typeahead_equals(":la", emoji_list);
     assert_typeahead_equals(" :lee", emoji_list);
+    assert_typeahead_equals("hi :see no", emoji_list);
+    assert_typeahead_equals("hi :japanese post of", emoji_list);
```

Comment:
Ahh, yeah, that seems to be the case; it's testing what sequences have the typeahead open.  Cool.  I'm glad you added the test below for what options are presented.